vit/command_bar.py

```diff
@@ -59,8 +45,20 @@ def keypress(self, size, key):
             return None
         elif key in self.readline.keys():
             return self.readline.keypress(key)
+        elif key in ('backspace',) and self.abort_backspace and not self.get_edit_text():
+            self.quit({'key': key})
+            return None
         return super().keypress(size, key)
 
+    def quit(self, metadata_args={}):
+        self.set_caption('')
+        self.set_edit_text('')
+        self.history.cleanup(self.metadata['op'])
+        data = {'metadata': self.get_metadata(), **metadata_args}
```

Comment:
Instead of ditching the ```cleanup()``` method, let's keep that and call it after getting ```data``` in ```quit()``` -- that seems a better separation of concerns.

Reply:
It was like that before, but I removed the `cleanup()` because it would trigger an exception when used as the only step of processing backspace, and thereby confused me what exactly it intends to do. I would suggest to document/rename it better.

Reply:
Well, that method handles all of the tasks of restoring things to a pristine state, and that does seem like a rational grouping of tasks for a method. Do you have any suggestions on a more clear name?

Reply:
Maybe `deactivate`?